heartbeat/portblock

```diff
@@ -289,24 +311,24 @@ tickle_local()
 	f=$OCF_RESKEY_tickle_dir/$OCF_RESKEY_ip
 	[ -r $f ] || return
 
-	checkcmd="netstat -tn"
-	if ! have_binary "netstat"; then
-		checkcmd="ss -Htn"
-	fi
-
 	# swap "local" and "remote" address,
 	# so we tickle ourselves.
 	# We set up a REJECT with tcp-reset before we do so, so we get rid of
 	# the no longer wanted potentially long lived "ESTABLISHED" connection
 	# entries on the IP we are going to delet in a sec.  These would get in
 	# the way if we switch-over and then switch-back in quick succession.
+
+	# Should we just use "ss --kill", with a filter on OCF_RESKEY_ip?
+	# Which old distributions do not have that yet?
```

Comment:
I guess easier to leave it as is.

Reply:
Let's leave it as is.

Reply:
I'll remove the comment then, and do an other force push in a minute.